packages/mgt-components/src/graph/graph.user.ts

```diff
@@ -310,10 +310,14 @@ export async function getUsersForPeopleQueries(
         const person = JSON.parse(cacheRes.results[0]);
         people.push(person);
       } else {
-        batch.get(personQuery, `/me/people?$search="${personQuery}"`, ['people.read']);
+        batch.get(personQuery, `/me/people?$search="${personQuery}"`, ['people.read'], {
```

Comment:
I really dislike how GitHub doesn't let me comment on lines that are unchanged :|
Anyhow. 
We can clean this logic up and de-duplicate the batch.get call here by removing a level of nesting and making the ` if (getIsUsersCacheEnabled()) {` block on line 307 a stand alone if check and not wrapping the whole block from 309 - 316 inside it.

Reply:
Something like:
```javascript
cacheRes = await cache.getValue(personQuery);
if (getIsUserCacheEnabled() && cacheRes?.results[0] && getUserInvalidationTime() > Date.now() - cacheRes.timeCached){
  const person = JSON.parse(cacheRes.results[0]);
  people.push(person);
}else{
  batch.get();
}

```